src/NativeScript/inlineFunctions.js

```diff
@@ -14,6 +14,103 @@ Object.assign(global, {
     NSMakeRange(location, length) {
         return new NSRange({ location, length });
     },
+
+    // Decorators support    
+    __decorate(decorators, target, key, desc) {
```

Comment:
Yes, they are the same, but with formatting and also changed `Reflect` to be `global.Reflect` as I saw this is what you use in the android runtime. I will change to follow the formatting of the TS default ones.